### PR TITLE
Integer vs. Floats_editorial review complete_KB to review

### PR DESCRIPTION
**Pull request by lizwar**
Before submitting a pull request for a new Learning Path, please review [Create a Learning Path](https://learn.arm.com//learning-paths/cross-platform/_example-learning-path/)
- [ x] I have reviewed Create a Learning Path

Please do not include any confidential information in your contribution. This includes confidential microarchitecture details and unannounced product information.
- [ x] I have checked my contribution for confidential information

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the Creative Commons Attribution 4.0 International License.

### CODE REVIEW

File: content/learning-paths/cross-platform/integer-vs-floats/type-demotion-problems.md [hunk 1]
```diff
@@ -78,12 +78,12 @@ a = -16380, b = -16380000, c = 799869184, d1 = 4608, d2 = 4608
 e1 = 2147483647, e2 = 2147483647
 ```
 
-Obviously `w` is a huge value and it does not fit in a `float` (remember largest positive float is `3.4e+38`), but the compiler only complains about `z` which uses bracket initialization and not `y` which uses assignment. 
+Obviously `w` is a huge value and it does not fit in a `float` (remember that the largest positive float is `3.4e+38`), but the compiler only complains about `z` which uses bracket initialization and not `y` which uses assignment. 
 
-Similarly, `d1` an `int16_t` uses assignment and does not generate a warning, but `d2`, also an `int16_t`, uses bracket initialization and trigger a compiler warning.
+Similarly, `d1` and `int16_t` use assignment and do not generate a warning but `d2`, also an `int16_t`, use bracket initialization and trigger a compiler warning.
```
Comment (markos) [inline content/learning-paths/cross-platform/integer-vs-floats/type-demotion-problems.md:83]: this actually denotes that `d1` is of type `int16_t`, not that there are two entities.

File: content/learning-paths/cross-platform/integer-vs-floats/introduction-integer-float-types.md [hunk 2]
```diff
@@ -118,42 +116,42 @@ x = 99999.98437500000000, 0x47c34ffe
...
+A similar point can be made for doubles but because the precision is twice that of 32-bit floats, you get 14 decimal digits of precision.
 
 {{% notice Summary %}}
 You are not dealing with real numbers, but approximate representations of real numbers using limited precision data types. 
 
 If you assign a real number value to a float data type, the result might not be the same real number you assigned.
 {{% /notice %}}
 
-If your calculations require full precision, then you should be looking at arbitrary precision libraries such as [The GNU Multiple Precision Arithmetic Library](https://gmplib.org/). 
+If your calculations require complete precision, then you should be looking at arbitrary precision libraries such as [The GNU Multiple Precision Arithmetic Library](https://gmplib.org/).
```
Comment (markos) [inline content/learning-paths/cross-platform/integer-vs-floats/introduction-integer-float-types.md:132]: either full precision is used or absolute precision as terminology in such contexts.

Comment (lizwar) [reply content/learning-paths/cross-platform/integer-vs-floats/introduction-integer-float-types.md]: Thanks. Will revert back to full

File: content/learning-paths/cross-platform/integer-vs-floats/type-demotion-problems.md (hunk 1 above)
Comment (lizwar) [reply content/learning-paths/cross-platform/integer-vs-floats/type-demotion-problems.md]: reverted back

**Pull request merged**